app.js

```diff
@@ -0,0 +1,59 @@
+	// create the module and name it oppia
+	var oppia = angular.module('oppiaApp', ['ngRoute']);
```

Comment:
Can we name this app oppiaGithubPages or something that's not 'oppia'? We use 'oppia' to identify the app in the main codebase, so if we use it here as well it would be confusing I think.

Also the variable name should probably follow the module name -- so the module name should also be oppiaGithubPages.